website/docs/tutorial/1-setting-up.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: Setting up
+title: Get started with dbt
 id: setting-up
+description: "Create your first dbt project using a SQL query."
 ---
 
-In this tutorial, we will be turning the below query into a dbt project that is tested, documented, and deployed — you can check out the generated documentation for the project we're building [here](https://www.getdbt.com/getting-started-tutorial/#!/overview?g_v=1).
+Learn how to turn the following SQL query into a tested, documented, and deployed dbt project. You can preview [the generated documentation](https://www.getdbt.com/getting-started-tutorial/#!/overview?g_v=1) for the project we're building. You'll need a working knowledge of SQL to complete this tutorial.
 
-This tutorial is geared at first-time users who want detailed instructions on how to go from zero to a deployed dbt project. You'll need a working knowledge of SQL in order to do this tutorial.
+If you want a more in-depth learning experience, consider taking some [dbt Learn online courses](https://courses.getdbt.com/).
```

Comment:
appreciation: appreciate testing out this callout coming out of our community <> training sync yesterday.